### ISSUE TITLE
Teslium doesn't think Syndicates are real people

### ISSUE THREAD

**Issue by extramrdo** [FTL13/New-FTL13#806]
## Reproduction:
Grow glowcaps with the traits Separated Chemicals, Water production, Teslium production, and Liquid Contents. Result is a plant that, when thrown, creates an electric shock to the nearest person. To quote the tg secondary guide to Hydroponics, "When thrown it will release a thunderbolt to the nearest target, shocking it and usually making it fall down." I confirmed that this does occur; this is not where the error occurs.

Board syndicate ship. Encounter syndicate enemy wielding sword, energy shield, armor vest, etc. Throw teslium grenade glowcap at wall. Syndicate is 1 tile away from impact. Self is at least 4 tiles away. Clear line of sight from impact to both.

Expected result: Lightning arcs from glowcap impact to syndicate enemy, because it is the "nearest target," inflicting damage.
Observed result: Lightning arcs from glowcap impact to self, inflicting damage to self.

In short, electric shocks don't see Syndicate mobs as valid targets.

**Comment by Dennok** [comment]
Send this to base TG. Like a "Lightning arcs from glowcap dont impact to simplemobs"
PS. Base old FTL is outdated, it can be already fixed.
